src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LoggingKeys.cs

```diff
@@ -1,32 +1,101 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using Microsoft.Extensions.Logging;
+
 namespace Microsoft.Azure.WebJobs.Host.Loggers
 {
-    internal static class LoggingKeys
+    /// <summary>
+    /// Keys used by the <see cref="ILogger"/> infrastructure.
+    /// </summary>
+    public static class LoggingKeys
     {
-        // These are publicly visible as property names or prefixes
+        /// <summary>
+        /// </summary>
         public const string FullName = "FullName";
+
+        /// <summary>
```

Comment:
I commented above that they were coming -- but then I forgot them :-) 

I've already got a change coming with namespace updates recommended by @mathewc and I've filled these in as well.